ink-wrapper/src/main.rs

```diff
@@ -448,3 +467,14 @@ fn resolve(metadata: &InkProject, id: u32) -> &Type<PortableForm> {
         .resolve(id)
         .unwrap_or_else(|| panic!("Type {} not found", id))
 }
+
+/// Generates a name not already used by one of the arguments.
+fn new_name(name: &str, args: &[MessageParamSpec<PortableForm>]) -> String {
+    let mut name = name.to_string();
+
+    while args.iter().any(|arg| arg.label() == &name) {
+        name.push('_');
+    }
```

Comment:
Do you need to do that? You're limiting the scope of these variables anyway. See the example below.

Reply:
Ah, you do need it - the `conn` and `conn_` is a proof.

Reply:
:+1: